### DISCUSSION TITLE
is there a benchmark for graph queries with apache/age

### DISCUSSION THREAD

**Discussion by mstrYoda** [authzed/spicedb#1876] [Q&A]
[Age](https://github.com/apache/age) is a Postgresql extension for graph queries. Did you make any benchmark for current sql queries vs graph queries using Age?

It might be a performance improvement (or not) 🤔

**Comment by josephschorr** [comment]
Datastores in SpiceDB are treated as simple versioned relationship stores; no JOINs or more complex forms of querying are used, which means it would be quite difficult to test.

Furthermore, since most such extensions are unsupported on cloud databases, its unlikely it would be easy or even possible to use
